app/components/sign-up.jsx

```diff
@@ -147,7 +147,7 @@ function SignUp(props, ref) {
         >
           <p id="text">Last Name</p>
           <input
-            name="last-name"
+            name="last-name" placeholder="Doe"
```

Comment:
Please install prettier for visual studio code: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

We already have a default setup for prettier, and it would separate these props into separate lines.  And it won't affect your other projects unless you configure it to do so.

One advantage of prettier, is there are less unexpected differences when diffing the files that people check in.